src/backend/commands/explain.c

```diff
@@ -2668,6 +2668,8 @@ show_sort_info(SortState *sortstate, ExplainState *es)
 	if (!es->analyze)
 		return;
 
+	Insist(((PlanState *) sortstate)->instrument);
```

Comment:
see above
https://github.com/arenadata/gpdb/blob/fd5939d4f491238bbd28be88b98949cd0e2d6fe0/src/backend/commands/explain.c#L2177

Reply:
What do you want to say by this?

Reply:
planstate->instrument may be NULL and it is OK

Reply:
Fixed

Reply:
update PR description according to changes

Reply:
Done